test/e2e/specs/specs-calypso/wp-editor__edit-media-spec.js

```diff
@@ -24,6 +25,11 @@ describe( `[${ host }] Media: Edit Media (${ screenSize }) @parallel @jetpack`,
 	} );
 
 	it( 'Select a random media item and click edit', async function () {
+		if ( process.env.FLAGS === 'nav-unification/switcher' ) {
```

Comment:
Who set this flag? AFAIK is not set in CI, so this will never run in CI.

Reply:
Thanks! Didn't know if it was set [here](https://github.com/Automattic/wp-calypso/blob/trunk/config/test.json#L99) but have been unable to run these tests locally so was seeing how they'd perform here. Will remove this flag.

Reply:
e2e tests use `calypso.live` links, which use the config from `wpcalypso.json` (I think, not 100% sure sure)